packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -815,11 +894,41 @@ export function BudgetTable(props) {
   let currentMonth = monthUtils.currentMonth();
   let format = useFormat();
 
+  const toggleMobileDisplayPref = useSelector(state => {
+    return (
+      (state.prefs.local && state.prefs.local.toggleMobileDisplayPref) || true
+    );
+  });
+
+  let [showBudgetView, setshowBudgetView] = useState(
+    !toggleMobileDisplayPref &&
+      !document.cookie.match(/toggleMobileDisplayPref=true/),
+  );
+
+  let dispatch = useDispatch();
+
+  function toggleDisplay() {
+    setshowBudgetView(!showBudgetView);
+    if (!showBudgetView) {
+      // remember the pref indefinitely
+      dispatch(savePrefs({ toggleMobileDisplayPref: true }));
+    } else {
+      // Set a cookie for 5 minutes
+      let d = new Date();
+      d.setTime(d.getTime() + 1000 * 60 * 5);
+      document.cookie =
+        'toggleMobileDisplayPref=true;path=/;expires=' + d.toUTCString();
+    }
```

Comment:
I'd say it's safe to drop the `else` altogether and just indefinitely show the column the person picked last. I do think it's a nice touch to remember it for them. 👏🏻